packages/core/review-workflows/admin/src/routes/content-manager/model/id/components/AssigneeSelect.tsx

```diff
@@ -19,7 +19,10 @@ import { getDisplayName } from '../../../../../utils/users';
 
 import { ASSIGNEE_ATTRIBUTE_NAME } from './constants';
 
-const AssigneeSelect = () => {
+type InjectionZoneArea = 'preview.actions';
+
+const AssigneeSelect = ({ area }: { area?: InjectionZoneArea }) => {
+  const isCompact = area === 'preview.actions';
```

Comment:
I assume there's enough complexity in there that you'd rather do this + pass the area prop to every injected component rather than have 2 versions of this component?

Reply:
Why not indeed. It would be easy easy to fix anyway if we get the report that it somehow broke something